app/admin/page.tsx

```
@@ -0,0 +1,13 @@
+export const dynamic = 'force-dynamic';
+export const dynamicParams = true;
+export const revalidate = 0;
+export const dynamicParams = true;
```

Biome (2.1.2)

[error] Shouldn't redeclare 'dynamicParams'. Consider to delete it or rename it.

'dynamicParams' is defined here:


(lint/suspicious/noRedeclare)